chance.js

```diff
@@ -1685,7 +1685,14 @@
 
     Chance.prototype.address = function (options) {
         options = initOptions(options);
-        return this.natural({min: 5, max: 2000}) + ' ' + this.street(options);
+        return this.random() < 0.0001 ? this.natural({min: 1, max: 999999}) + ' ' + this.street(options) : 
```

Comment:
Can I recommend using `this.weighted()` as a cleaner approach?

``` javascript
// favor lower civic numbers
const maxCivicNumber = this.weighted( [2000, 5_000, 10_000, 25_000, 50_000, 100_000, 500_000], [1600, 800, 400, 200, 20, 5, 1]);
return this.natural({min: 1, max: maxCivicNumber})  + ' ' + this.street(options);
```

Reply:
Yes, I agree with this, since the probability calculations are close to my implementation.